js/ui/components/clubhouse.js

```diff
@@ -645,13 +645,31 @@ class ClubhouseNotificationSource extends NotificationDaemon.GtkNotificationDaem
     }
 };
 
+// Enabling hack-mode for hack1 users
+function _migrateHack1() {
+    const hackComponents = Gio.File.new_for_uri('file:///var/lib/flatpak/app/com.endlessm.HackComponents');
+    if (!hackComponents.query_exists(null))
+        return;
+
+    // Only enable the first time, to allow the user to disable the hack mode
+    try {
+        Gio.File.new_for_path('.hack1-migrated').create(Gio.FileCreateFlags.NONE, null);
```

Comment:
Surely, surely this should be relative to $XDG_CONFIG_DIR or similar, rather than just "in whatever the current working directory happens to be"?

Reply:
I've just created a new PR to use the user config dir: https://github.com/endlessm/gnome-shell/pull/597